### PR TITLE
Reorg summary.md

### PR DESCRIPTION
**Pull request by DebbieAtSeam**
- Just reorg summary.md.
- Change text of some table of contents (ToC) entries, but _not_ filenames or file locations.
- Add two needed additional topics to serve as parents of existing topics.
- All other changes should be handled in separate PRs after this one.

(We need different, better emojis for the newly-added major sections, but I think it might be safer to change them in GitBook, rather than through GitHub.)

### CODE REVIEW

**Review by seveibar — APPROVED** [review]

**Review by sybohy — APPROVED** [review]

**Comment by sybohy** [comment]
LGTM! I saw you nested the getting started guides without changing the URL. I kind of like it 😄

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -18,35 +19,58 @@
   * [Managed and Unmanaged Devices](core-concepts/devices/managed-and-unmanaged-devices.md)
   * [Adding Custom Metadata to a Device](core-concepts/devices/adding-custom-metadata-to-a-device.md)
   * [Filtering Devices by Custom Metadata](core-concepts/devices/filtering-devices-by-custom-metadata.md)
+* [Connected Accounts](core-concepts/connected-accounts/README.md)
+  * [Adding Custom Metadata to a Connected Account](core-concepts/connected-accounts/adding-custom-metadata-to-a-connected-account.md)
+  * [Filtering Connected Accounts by Custom Metadata](core-concepts/connected-accounts/filtering-connected-accounts-by-custom-metadata.md)
+  * [Reconnecting an Account](device-guides/reconnecting-an-account.md)
+* [Action Attempts](core-concepts/action-attempts.md)
+
+## 🧰 UI Components
```
Comment (dawnho) [inline docs/SUMMARY.md:28]: I would move the `Capability Guides` section above `UI Components`

**Review by dawnho — APPROVED** [review]
Small comment about ordering. Otherwise, looks good!

File: docs/SUMMARY.md (hunk 1 above)
Comment (DebbieAtSeam) [reply docs/SUMMARY.md:28]: @dawnho and @sybohy: Looks like you might not be in agreement about the order of the UI Components section vs. the Capability Guide section. Happy to order them either way, as long as you two agree. Thanks!

Comment (DebbieAtSeam) [reply docs/SUMMARY.md]: Moved.

**Pull request merged**